site/en/docs/extensions/reference/tabs/index.md

```diff
@@ -24,17 +24,79 @@ permission in the manifest, as shown below:
 
 ## Examples
 
-![Two tabs in a window](tabs.png)
+The following sections demonstrate several common use cases for the chrome.tabs API.
 
-You can find simple examples of manipulating tabs with the `chrome.tabs` API in the
-[examples/api/tabs][7] directory. For other examples and for help in viewing the source code, see
-[Samples][8].
+### Opening an extension page in a new tab
 
-[1]: /docs/extensions/mv2/tabs
-[2]: #property-Tab-url
-[3]: #property-Tab-pendingUrl
-[4]: #property-Tab-title
-[5]: #property-Tab-favIconUrl
-[6]: #type-Tab
-[7]: https://github.com/GoogleChrome/chrome-extensions-samples/tree/master/mv2-archive/api/tabs/
-[8]: /docs/extensions/mv2/samples
+A common pattern for extensions is to open an onboarding page in a new tab when the extension is
+installed. The following example shows how to do this.
+
+{% Aside %}
+Cannot use `chrome.tabs.create()` in content scripts.
```

Comment:
Is a sentence with no subject. Consider any of:

- Content scripts cannot use `chrome.tabs.create()`
- `chrome.tabs.create()` is not available in content scripts.
- The `chrome.tabs.create()` method is not available in content scripts.